package.json

```diff
@@ -84,16 +84,16 @@
     "xterm-addon-fit": "0.4.0"
   },
   "devDependencies": {
-    "eslint-config-airbnb": "18.1.0",
+    "eslint-config-airbnb": "18.2.0",
```

Comment:
I don't think we actually use this any longer, we'll have to look at removing it.